ndsl/dsl/dace/stree/optimizations/axis_merge.py

```diff
@@ -0,0 +1,438 @@
+from __future__ import annotations
+
+import copy
+import re
+from typing import Any, List
```

Comment:
Let's use lowercase `list` in new code

```suggestion
from typing import Any
```